guides/elements.md

```diff
@@ -562,3 +562,49 @@ You can use the icons to copy and cut a specific element from its expanded state
 You can view the contents of the clipboard using the icon at the top of the elements bar.
 
 When adding a new top-level element, the "Paste from clipboard" tab will be visible if the clipboard is storing any elements.
+
+
+## Deprecating an element
+
+Occasionally, an element needs to be removed, either because it is no
+longer in use or is being replaced by another.
+Alchemy provides a straightforward deprecation system to notify editors of such changes.
+
+### Mark an element as deprecated
+
+Add `deprecated: true` to the element definition to mark it as deprecated:
+~~~ yaml
+- name: hero_carousel_element
+  deprecated: true
+~~~
+
+By default, a deprecation notice will be displayed with the following message:
+
+```WARNING! This element is deprecated and will be removed soon. Please do not use it anymore.```
```

Comment:
This still looks a bit weirdly formatted

![Elements | AlchemyCMS 2025-02-05 21-42-45](https://github.com/user-attachments/assets/a61459b5-0595-4fc1-b9b5-cbc4243273f5)

Maybe using a block quote looks better?

```suggestion
> WARNING! This element is deprecated and will be removed soon. Please do not use it anymore.
```